coff.ml

```diff
@@ -1038,31 +1038,34 @@ end
 
 
 module Stacksize = struct
-  let set_stack_reserve filename reserve =
+  let set_stack_reserve filename size_of_stack_reserve =
     let filename =
       if not (Sys.file_exists filename) && (Sys.file_exists (filename ^ ".exe")) then filename ^ ".exe"
       else filename
     in
     let ic = open_in_bin filename in
-    let hdr_offset = int16 (read ic 0x3c 2) 0 in
-    let pe_signature = read_str ic hdr_offset 4 in
+    let pe_signature_offset = int32 (read ic 0x3c 4) 0 |> Int32.to_int in
```

Comment:
We have an aptly named function for that, and we can add a primary reference to the field (given that the PE spec doesn't state its size):
```suggestion
    (* Read e_lfanew from the IMAGE_DOS_HEADER (see winnt.h) *)
    let pe_signature_offset = int32_ (read ic 0x3c 4) 0 in
```